diff --git a/chromium/content/browser/BUILD.gn b/chromium/content/browser/BUILD.gn
index ce2c7bc8d9..c19abeda06 100644
--- a/chromium/content/browser/BUILD.gn
+++ b/chromium/content/browser/BUILD.gn
@@ -61,6 +61,7 @@ jumbo_static_library("devtools_protocol") {
 
   deps = [
     "//base",
+    "//components/spellcheck:buildflags",
     "//content/browser/devtools:devtools_background_services_proto",
     "//content/browser/devtools:protocol_sources",
     "//content/common:buildflags",
